sql/updates/world/3.3.5/2018_01_17_xx_world.sql

```diff
@@ -0,0 +1,6 @@
+-- Reanimated Abomination Abilities
+UPDATE `creature_template` SET `spell1` = 59564, `spell2` = 59576 WHERE `entry` = 31692;
+
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_q13264_thats_abominable';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+	(59576, 'spell_q13264_thats_abominable');
```

Comment:
still missing newline.

Reply:
You mean it should be like this?
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_q13264_thats_abominable';

INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
(59576, 'spell_q13264_thats_abominable');
```

Reply:
Looks good to me, just remove the blank line between the `DELETE FROM` & `INSERT INTO` queries.
TC codestyle suggests to add a blank comment line on top: `--  `